rustybeer/src/yeast/res/yeast.txt

```diff
@@ -0,0 +1,123 @@
+company	name	id	attenuation	flocculation	opt_temperature	alc_tolerance
+Wyeast	German Ale	1007	73-77%	Low	55-68 °F	11%
```

Comment:
I would prefer json as it's already used for beer styles and hops. Also using csv introduces additional dependency. What do you think?

Reply:
Mainly I wanted to try out other type of parsing as an exercise. But I also think data listings are better as tabular data, since there's a lot less fieldname and syntax noise and can be utilized with e.g. awk :grinning:

Reply:
Ok 😊 JSON just works a bit better with the rustybeer REST API (rustybeer-aws-lambda project)

Reply:
ah, didn't know about this. I can change it.